src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -2253,6 +2253,103 @@ public function get_test_authorization_header() {
 		return $result;
 	}
 
+	/**
+	 * Tests persistent_object_cache health check.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @return array The health check result suggesting persistent object caching.
+	 */
+	public function get_test_health_persistent_object_cache() {
```

Comment:
See my other comment:
```suggestion
	public function get_test_persistent_object_cache() {
```